packages/client/components/DiscussionThreadList.tsx

```diff
@@ -123,6 +123,9 @@ export default createFragmentContainer(DiscussionThreadList, {
   threadables: graphql`
     fragment DiscussionThreadList_threadables on Threadable @relay(plural: true) {
       ...ThreadedItem_threadable
+      ... on Poll {
+        updatedAt
+      }
```

Comment:
This is needed here so it triggers `useScrollThreadList` when new poll option is added

Reply:
It has to be here due to the fact that Relay masks the data. The hook `useScrollThreadList` will not see any changes in `threadables` unless this component specifies this data dependency.